server/Dockerfile

```diff
@@ -25,13 +25,15 @@ RUN pip install --upgrade pip
 
 # install dependencies
 COPY ./requirements.txt .
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /usr/src/app/wheels -r requirements.txt
+RUN pip wheel --no-cache-dir --no-deps --wheel-dir /usr/src/app/wheels -r requirements.txt  
 
+# TODO: this is a workaround, to be replaced when working out a proper data migration approach, possibly
+# when using entrypoint.prod.sh to seed dat, need dev requirements installed. Also needed for testing.
 # TODO: this is a workaround, to be replaced when working out a proper data migration approach, possibly
 # when using entrypoint.prod.sh to seed dat, need dev requirements installed. Otherwise, leave this commented out
```

Comment:
Comment duplicated?

Reply:
Do I put commit tags in here? Realizing my commit comments need to be more descriptive: [removed douplicate comment per Stephen's review](https://github.com/PHACDataHub/cpho-phase2/pull/73/commits/189b4e19d4725e0303b6fe3595d7af9b06779572) [189b4e1](https://github.com/PHACDataHub/cpho-phase2/pull/73/commits/189b4e19d4725e0303b6fe3595d7af9b06779572)

Reply:
Ooo, I'd appreciate if you do! Usually, either when I see commits to a branch I'm reviewer on or when prompted for re-review, I just look for comments that GitHub has marked "outdated" (meaning some line of code in the comment's scope has since changed) and try to figure out my self whether it was resolved or not, haha.

I always consider it the reviewer's responsibility to follow up and hit the resolve button, but I don't know if everyone operates that way.